src/controllers/worldstate.js

```diff
@@ -12,7 +12,28 @@ const get = (platform, language) => {
   }
 };
 
-const router = express.Router();
+const redirectCheck = (req, res) => {
+  if (req.platform !== 'pc') {
+    const redirPath = req.originalUrl
+      .replace(/\/(ps4|psn|swi|xb1|ns)/gi, '/pc')
+      .replace(/\/(ps4|psn|swi|xb1|ns)\//gi, '/pc/');
```

Comment:
I can't see a scenery where the second replace would be used, by my tests it's safe to remove it.

Reply:
If we think regexes as sets the first one is a subest of the second, so all that the second matches the first one will match it first.

Reply:
they _should_ work that way, but i couldn't make them work that way in mine. do you have one that worked locally for you?

Reply:
Well as this is something temporary and it's just technically dead code, i don't see much of a problem.

Reply:
true, but i'll probably leave this in and do a separate check for PC -> ws when the time comes, or just redir everything to ws when the time comes